decidim-proposals/app/controllers/decidim/proposals/collaborative_drafts_controller.rb

```diff
@@ -144,9 +144,9 @@ def search_collection
       def default_filter_params
         {
           search_text_cont: "",
-          with_any_category: default_filter_category_params,
+          with_any_category: nil,
```

Comment:
check the comment on decidim-budgets/app/controllers/decidim/budgets/projects_controller.rb